stdlib/public/core/Builtin.swift

```diff
@@ -730,13 +730,7 @@ internal func _unsafeDowncastToAnyObject(fromAny any: Any) -> AnyObject {
                "Any expected to contain object reference")
   // With a SIL instruction, we could more efficiently grab the object reference
   // out of the Any's inline storage.
```

Comment:
We have the ability to do what this comment is referencing now, in fact. Instead of `any as AnyObject`, this should be more efficient:

```
  _sanityCheck(type(of: any) is AnyObject.Type
                || type(of: any) is AnyObject.Protocol,
                 "Any expected to contain object reference")
  func open<T>(x: T) -> AnyObject {
    return unsafeBitCast(x, to: AnyObject.self)
  }
  return _openExistential(any, do: open)
```